lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeBootstrapConfigurationGenerator.java

```diff
@@ -16,13 +16,22 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+
 package org.apache.reef.runtime.multi.client;
 
-import org.apache.reef.tang.formats.ConfigurationModuleBuilder;
+import org.apache.reef.tang.Configuration;
+import org.apache.reef.tang.annotations.DefaultImplementation;
 
 /**
- * A ConfigurationModule to bind arbitrary named parameters for runtimes.
+ * Defines a contract for a multi runtime bootstrap configuration.
+ * This generator adds configureations that are needed for the multiruntime to run, s
+ * uch as classpath provider etc.
  */
-final class MultiRuntimeHelperConfiguration extends ConfigurationModuleBuilder {
+@DefaultImplementation(EmptyMultiRuntimeBootstrapConfigurationGeneratorImpl.class)
+interface MultiRuntimeBootstrapConfigurationGenerator {
```

Comment:
I actually removed the MultiRuntimeHelperConfiguration class, as it was just an empty one and replaced it with using TAng.Factory.newConfigurationModule. MultiRuntimeBootstrapConfigurationGenerator is a completely new file. Not sure why git decided that this is a file change.